channels/pe.m3u

```diff
@@ -64,3 +64,5 @@ https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="" tvg-logo="https://graph.facebook.com/BestCableTvCanal3/picture?width=320&height=320" group-title="",Best Cable Perú
 #EXTVLCOPT:http-referrer=https://bestcableperu.com.pe/tv3.html
 https://live.siete.us/bestcable/bestcable/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="" tvg-logo="https://i.imgur.com/zYpMHhQ.png" group-title="Local",Millenium 49 TV (Pucallpa)
+https://stmv.panel.mivideo.pro/mileniuntv/mileniuntv/playlist.m3u8
```

Comment:
Doesn't work for me -> 404 Not Found.

Reply:
The channel is https://millenium49tv.com/, because this website opens 8:00-23:00, I tagged [Not 24/7]

Reply:
Noted.